backend/src/main/java/com/huellapositiva/infrastructure/orm/repository/JpaCredentialRepository.java

```diff
@@ -33,6 +34,14 @@
     @Query("UPDATE JpaCredential c SET c.hashedPassword = :hash WHERE c.email = :email")
     Integer updatePassword(@Param("hash") String hash, @Param("email") String email);
 
+    @Modifying
+    @Transactional
+    @Query("UPDATE JpaCredential c SET c.name = :name, c.surname = :surname, c.email = :emailChange," +
+            " c.phoneNumber = :phoneNumber, c.birthDate = :birthDate WHERE c.email = :email")
+    Integer updateProfile(@Param("email") String email, @Param("name") String name, @Param("surname") String surname,
+                          @Param("emailChange") String emailChange, @Param("phoneNumber") Integer phoneNumber,
+                           @Param("birthDate") LocalDate birthDate);
+
```

Comment:
Move to JpaProfileRepository

Reply:
This method no longer has any use. I have support in the ORM to do the changes in database.